release/_publishing.sh

```diff
@@ -1,5 +1,44 @@
 #!/bin/bash
 
+function add_fixed_issues_to_project_version {
+	IFS=',' read -r -a fixed_issues_array < "${_BUILD_DIR}/release/release-notes.txt"
+
+	local fixed_issues_array_length="${#fixed_issues_array[@]}"
+
+	local fixed_issues_array_part_length=$((fixed_issues_array_length / 4))
+
+	for counter in {0..3}
+	do
+		local start_index=$((counter * fixed_issues_array_part_length))
+
+		if [ "${counter}" -eq 3 ]
+		then
+			fixed_issues_array_part_length=$((fixed_issues_array_length - start_index))
+		fi
```

Comment:
@kiwm, why this is still necessary? Can you please clarify?

Reply:
The approach I'm using is necessary due to the way we need to distribute the tickets. Let's say we have a total of 10 tickets.
10/ 4 = 2 = fixed_issues_array_part_length
The first part will have 2 tickets.
The second part will also have 2 tickets.
The third part will again have 2 tickets.

For the last part, we need to handle any remaining tickets differently. Instead of following the fixed part length, we need to include all remaining tickets in the last part to ensure the total is 10.

I have to put 4 tickets because:
fixed_issues_array_length - start_index =
total - counter * fixed_issues_array_part_length =
10 - 3 * 2 = 4

Reply:
I see your point, @kiwm. Basically, you need to handle the scenario in which the division `fixed_issues_array_length / 4` doesn't provide an exact result every time, right?

How about we try this instead?
```
if [ "${counter}" -eq 3 && $((fixed_issues_array_length % 4)) -ne 0) ]
then
	fixed_issues_array_part_length=$((fixed_issues_array_length % 4))
fi
```

The purpose of this code is basically get the division remainder that it's exactly what you're looking for. :slightly_smiling_face: